### PR TITLE
docs(functions): reorg Context functions style

### PR DESCRIPTION
**Pull request by suyanhanx**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

part of #10029

### CODE REVIEW

File: docs/doc/15-sql-functions/90-context-functions/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: 'Array Functions'
```
Comment (b41sh) [inline docs/doc/15-sql-functions/90-context-functions/index.md:2]: Context Functions

Comment (suyanhanx) [reply docs/doc/15-sql-functions/90-context-functions/index.md:2]: Updated. Thanks.

File: docs/doc/15-sql-functions/90-context-functions/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------|-----------------------------------------------------------------------------------------|
+| **CONNECTION_ID()**          | Return the connection id for the connection.                                                                                                                                                                                     | **CONNECTION_ID()**                  | 9b16ca73-e393-43bb-a77b-fa546e1b7b69                                                    |
+| **current_user()**           | Returns the user name and host name combination for the account that the server used to authenticate the current client. This account determines your access privileges. The return value is a string in the utf8 character set. | **current_user()**                   | 'root'@'127.0.0.1'                                                                      |
```
Comment (b41sh) [inline docs/doc/15-sql-functions/90-context-functions/index.md:8]: This function name better to use capital letters like other functions.

Comment (suyanhanx) [reply docs/doc/15-sql-functions/90-context-functions/index.md:8]: I agree. I initiated a discussion in the tracking issue to address this.

Comment (suyanhanx) [reply docs/doc/15-sql-functions/90-context-functions/index.md]: Updated.

**Review by b41sh — APPROVED** [review]

**Pull request merged**